block/batch.go

```diff
@@ -35,3 +42,8 @@ func (bq *BatchQueue) Next() *BatchWithTime {
 	bq.queue = bq.queue[1:]
 	return &batch
 }
+
+// NotifyCh returns the notification channel
+func (bq *BatchQueue) NotifyCh() <-chan struct{} {
```

Comment:
is this actually called anywhere? I don't think I saw it used, in which case we should just remove it.